app/src/test/java/org/mozilla/fenix/settings/wallpaper/ExtensionsTest.kt

```diff
@@ -54,6 +54,102 @@ class ExtensionsTest {
         assertEquals(downloadedSeasonalWallpapers, result[seasonalCollection])
     }
 
+    @Test
+    fun `GIVEN two collections of enough size WHEN grouped for onboarding THEN result contains 3 seasonal and 2 classic`() {
```

Comment:
```suggestion
    fun `GIVEN two collections of appropriate size WHEN fetched for onboarding THEN result contains 3 seasonal and 2 classic`() {
```
s/enough/appropriate/g
s/not enough/insufficient/g

Reply:
@mavduevskiy Just a bump on this. This is mostly an English wording nit.

Reply:
I have updated to the suggested version, is there anything else I should do?

Reply:
`/g` meant global here so we'll want to do the same for the other tests.

Reply:
oh, got it! what does `/s` stand for?

Reply:
Sorry, `s/enough/appropriate/g` was a sed notation which is a linux command for search and transforming strings. You can read this as s/<old>/<new> where <old> will be replaced by <new> `/g` will be global and similar to what you see with regular expressions.

https://linuxhint.com/50_sed_command_examples/#s1

Reply:
thanks! I didn't understand your comment properly, I did get only the suggested part.
updated!

Reply:
@mavduevskiy Sorry looks like the `<old>` and `<new>` were hidden in the markdown. I updated the comment in case it will be helpful.